.gitignore

```diff
@@ -22,4 +22,4 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 .eslintcache
-src/config/config.ts
```

Comment:
It's checked in to the repo so figured we shouldn't pretend it's ignored

Reply:
I'm assuming it's ignored because it acted as an env-ish file, but I agree, let's not do this from now :+1: